server/db/initiatives-migration.sql

```diff
@@ -0,0 +1,165 @@
+DROP TABLE IF EXISTS initiative;
+-- Create Initiative Table
+CREATE TABLE initiative (
+   id serial PRIMARY KEY,
+   name VARCHAR(50) UNIQUE NOT NULL,
+   category VARCHAR(50) NOT NULL,
+   description VARCHAR(255) NOT NULL,
+   createdById INT NOT NULL,
+   createdAt TIMESTAMP NOT NULL DEFAULT NOW(),
+   updatedById INT NOT NULL,
+   updatedAt TIMESTAMP NOT NULL DEFAULT NOW()
```

Comment:
I don't think we need updated by, if we are adding updated by then we need to add the relevant reference
```suggestion
   updatedById INT NOT NULL,
   updatedAt TIMESTAMP NOT NULL DEFAULT NOW()
   FOREIGN KEY (updatedById) REFERENCES users (id)
   FOREIGN KEY (createdById) REFERENCES users (id)
```